modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -191,16 +191,18 @@ public CompletableFuture<Void> start() {
 
         updateLog.registerUpdateHandler(new OnUpdateHandlerImpl());
 
-        updateLog.start();
+        updateLog.startAsync();
```

Comment:
That's not part of your PR actually, but anyway, we should not ignore `updateLog.startAsync();` future. Proper async postfix 've made it visible.